### PR TITLE
Adds overview pages

### PR DESCRIPTION
**Pull request by jmsmkn**
This PR adds overview pages that can be used to add extra items to the nav bar, linking together archives, algorithms, reader studies and challenges. It has a user customisable markdown at the top of the page, followed by cards:

Screenshot_20200330_154029

### CODE REVIEW

File: app/grandchallenge/core/templates/grandchallenge/partials/navbar.html [hunk 1]
```diff
@@ -31,6 +31,14 @@
                         Reader Studies
                     </a>
                 </li>
+                {% for overview_page in overview_pages %}
+                    <li class="nav-item">
+                        <a class="nav-link {% if request.resolver_match.view_name == 'overview-pages:detail' %}active{% endif %}"
```
Comment (MikeOverkamp-diag) [inline app/grandchallenge/core/templates/grandchallenge/partials/navbar.html]: Wouldn't this make all nav items active on all overview pages?

File: app/grandchallenge/overview_pages/admin.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    class Meta:
+        model = OverviewPage
+        widgets = {
+            "detail_page_markdown": MarkdownEditorAdminWidget,
+        }
+        exclude = ("description",)
+
+
+class OverviewPageAdmin(MarkdownxModelAdmin):
+    can_add_related = False
```
Comment (MikeOverkamp-diag) [inline app/grandchallenge/overview_pages/admin.py]: Does this do something different from the widget keyword?

Comment (jmsmkn) [reply app/grandchallenge/overview_pages/admin.py]: Nope, left over by mistake, thanks!

File: app/grandchallenge/core/templates/grandchallenge/partials/navbar.html (hunk 1 above)
Comment (jmsmkn) [reply app/grandchallenge/core/templates/grandchallenge/partials/navbar.html]: Ah yes, we don't do anything with active in the navbar anyway so I will remove it

Comment (jmsmkn) [reply app/grandchallenge/core/templates/grandchallenge/partials/navbar.html]: Actually kept it by looking up slug too. Can't use the url checks as we have this complex url reversal for subdomains.

**Pull request merged**